model_compiler_2/src/model_compiler/models/sources/tf_model_file.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 ZTE corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+from os import path
+from typing import Any, Mapping, NamedTuple, Optional
+
+from .. import repository
+
+
+@repository.REPOSITORY.register_source_model
+class TfModelFile(NamedTuple):

```

Comment:
Split this model into two model types corresponds to `model_path` and `script_path`.